terra-jupyter-aou/Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN jupyter nbextension enable --py igv --sys-prefix
 ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
 
 ENV PYSPARK_PYTHON=python3
-ENV HAIL_VERSION=0.2.109
+ENV HAIL_VERSION=0.2.112
```

Comment:
Dan is already updating this in his PR https://github.com/DataBiosphere/terra-docker/pull/408

Reply:
I left a comment in that PR and let Dan know this PR also includes Hail upgrade